src/clearblade.c

```diff
@@ -48,9 +49,9 @@ void validateInitOptions(struct ClearBlade *CB) {
 }
 
 /**
-  * This function validates the initialize parameters and then calls the authentication function in user.c
+  * This function validates the user initialize parameters and then calls the authentication function in user.c
 */
-void initialize(struct ClearBlade *CB, void callback(bool error, char *result)) {
+void initializeUser(struct ClearBlade *CB, void callback(bool error, char *result)) {
```

Comment:
I think we should not change func names. Some of our customers use this SDK and we want to be backwards compatible